lib/overcommit/hook/pre_commit/base.rb

```diff
@@ -1,79 +1,11 @@
 require 'forwardable'
+require 'overcommit/utils/messages_utils'
```

Comment:
@sds ,

I have a question on this file loading. It is possible to avoid defining this and use autoloading instead? I notice modules like GitRepo and FileUtils we can use them via autoloading.

Reply:
As we are not using autoloading anywhere else in this project, I would prefer not to introduce it at this time. I've been bitten by odd loading errors when trying to be too clever (usually not problems caused by autoloading itself, but by how others load a library locally). If someone uses your code in a way you didn't expect, they could trigger "autoloading" of classes at a time that you don't expect, which can cause difficult to debug issues.

Reply:
Noted.